autopilot/autopilot.go

```diff
@@ -53,9 +54,15 @@ type Bus interface {
 
 	// consensus
 	ConsensusState() (bus.ConsensusState, error)
+
+	// objects
+	MarkSlabsMigrationFailure(slabIDs ...bus.SlabID) (int, error)
```

Comment:
slice!

Reply:
also, what's the purpose of marking failed migrations?

Reply:
If repairs fail on a few slabs for some reason we want to mark them as failed so that they aren't returned in the next batch of slabs for migration until some time has passed.
Essentially it's supposed to make sure migrations don't get stuck on specific issues. Shouldn't happen in a perfect world but it might for random reasons.